### ISSUE TITLE
Fluid interaction crash: Incompatibility with `Plus Tweaks`

### ISSUE THREAD

**Issue by Krutoy242** [jbredwards/Fluidlogged-API#117]
**Describe the bug**
Im using [Plus Tweaks](https://www.curseforge.com/minecraft/mc-mods/plus-tweaks) mod and its custom fluid interactions:
```less
mods.plustweaks.Liquid.registerLiquidInteraction(ILiquidStack liquid1, ILiquidStack liquid2, IBlockState block, int[] levels);
```

Game crashes when registered liquid touches another fluidlogged block:

Game freezes after this:
<img src="https://user-images.githubusercontent.com/936618/184965541-d8f3a141-6826-4989-8c28-9e550f6d8a98.png" width=300>

Break glass to crash:
<img src="https://user-images.githubusercontent.com/936618/184965592-4a141830-a8b5-4d73-86e9-4483974827bc.png" width=300>

```less
java.lang.IllegalArgumentException: Cannot get property PropertyInteger{name=level, clazz=class java.lang.Integer, values=[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]} as it does not exist in BlockStateContainer{block=minecraft:cobblestone, properties=[]}
    at net.minecraft.block.state.BlockStateContainer$StateImplementation.getValue(BlockStateContainer.java:209)
    at net.minecraft.block.BlockLiquid.handler$zfo000$injectCheckForMixing(BlockLiquid.java:1115)
    at net.minecraft.block.BlockLiquid.checkForMixing(BlockLiquid.java)
    at net.minecraft.block.BlockDynamicLiquid.onBlockAdded(BlockDynamicLiquid.java:304)
    at net.minecraft.world.chunk.Chunk.setBlockState(Chunk.java:614)
    at git.jbredwards.fluidlogged_api.mod.asm.plugins.ASMHooks.setBlockState(ASMHooks.java:1598)
 ```

**To Reproduce**
1. Install `Fluidlogged-API-v1.8.1d-mc1.12.2`, PlusTweaks, CraftTweaker
2. Register Liquid Interaction (probably need other mods, because vanilla already have interactions)
3. Make Liquid from recipe touch another

**Logs**
[crash-2022-08-16_12.00.34-server.txt](https://github.com/CleanroomMC/Fluidlogged-API/files/9353947/crash-2022-08-16_12.00.34-server.txt)
[crash-2022-08-16_19.09.41-server.txt](https://github.com/CleanroomMC/Fluidlogged-API/files/9353948/crash-2022-08-16_19.09.41-server.txt)

**Additional context**
This is happens for players of [`E2E-E` modpack](https://www.curseforge.com/minecraft/modpacks/enigmatica-2-expert-extended)
Same issue [posted on `Plus Tweaks` tracker](https://github.com/misterplus/PlusTweaks/issues/2) too.

**Comment by jbredwards** [comment]
This issue is caused by PlusTweaks, it's trying to get the liquid level property from the block holding the fluidstate (rather than getting it from the fluidstate itself). Since it's been a while since that mod's last update (or even last commit for that matter), I'll use asm to fix it from my side until they add an official fix from their side.

**Comment by jbredwards** [comment]
fixed in v1.9.0.0
